app/src/main/java/org/sopt/and/data/api/AuthService.kt

```diff
@@ -0,0 +1,27 @@
+package org.sopt.and.data.api
+
+import org.sopt.and.data.dto.RequestLoginData
+import org.sopt.and.data.dto.RequestUserRegistrationData
+import org.sopt.and.data.dto.ResponseLogin
+import org.sopt.and.data.dto.ResponseUserRegistration
+import retrofit2.Response
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+
+interface UserRegistrationService {
+    @POST("/user")
+    suspend fun postUserRegistration(
+        @Body userRequest: RequestUserRegistrationData
+    ): Response<ResponseUserRegistration>
+}
+
+interface LoginService {
+    @POST("/login")
+    suspend fun postLogin(
+        @Body loginRequeset: RequestLoginData
+    ): Response<ResponseLogin>
+}
```

Comment:
AuthService 라는 이름에 맞춰서 다음처럼 하나의  인터페이스로 구현하는 건 어떤가요?
```suggestion
interface AuthService{
    @POST("/user")
    suspend fun postUserRegistration(
        @Body userRequest: RequestUserRegistrationData
    ): Response<ResponseUserRegistration>
    
     @POST("/login")
    suspend fun postLogin(
        @Body loginRequeset: RequestLoginData
    ): Response<ResponseLogin>
}

```